lakey_service/downloader/executors/databricks.py

```diff
@@ -1,13 +1,43 @@
+from databricks_api import DatabricksAPI
+from datetime import datetime
+from django.conf import settings
 
+from .baseexecutor import BaseExecutor
 
-class DataBricksExecutor:
-    # https://docs.databricks.com/api/latest/jobs.html#jobsnotebooktask
 
-    def start_cluster(self):
-        pass
+class DatabricksExecutor(BaseExecutor):
 
-    def start_job(self, spec):
-        pass
+    def __init__(self):
+        self.db = DatabricksAPI(
+            host=settings.DATABRICKS_HOST,
+            token=settings.DATABRICKS_TOKEN
+        )
 
-    def get_job_status(self, job_id):
-        pass
+    def __generate_file_path(self):
+        """ Return path of output file """
+        now = datetime.now()
+        output_location = settings.DATABRICKS_RESULTS_LOCATION
+        filename = now.strftime("%Y%m%d-%H%M%S")
+
+        return output_location+filename+"_output.csv"
+
+    def execute(self, download_request):
+
+        query = self.compile_to_query(download_request)
+
+        return self.execute_query(query)
+
+    def execute_query(self, query):
+        output_file = self.__generate_file_path()
+
+        self.db.jobs.submit_run(
+            run_name='Lakey get data',
```

Comment:
shouldn't we the run name be somehow randomized, you know in order to introduce some uniqueness?

Reply:
Do you see any specific point in doing that?
There is (by default) uniqueness in assigned run_id. Do we need run name to be unique as well?

Reply:
honestly I don't remember how runs are represented in the DataBricks UI / CLI so was just wondering if we would be able to distinguish jobs when listing them

Reply:
OK, so I'll leave it for now and we can recheck in practice during final integration with our dbricks.